src/richie/plugins/lti_consumer/models.py

```diff
@@ -41,6 +41,8 @@ class LTIConsumer(CMSPlugin):
     )
     oauth_consumer_key = models.CharField(max_length=50, null=True, blank=True)
     shared_secret = models.CharField(max_length=50, null=True, blank=True)
+    is_automatic_resizing = models.BooleanField(default=True, blank=True)
+    inline_ratio = models.FloatField(default=0.5625, blank=True)  # default is 16/9
```

Comment:
```suggestion
    inline_ratio = models.FloatField(default=0.5625, blank=True, min_value=0)  # default is 16/9
```

We could also add a `max_value` constraint but not sure about the value to use (3?)

Reply:
Good catch. Let me put 0.1 as mean value because a ratio of 0 doesn't make sense anyway.

Reply:
There's no `min_value` or `max_value` properties on `FloatField`. So I used validators and chose 0.1 < x < 10:
```
inline_ratio = models.FloatField(
    ...
    validators=[MinValueValidator(0.1), MaxValueValidator(10)]
)
```